### ISSUE TITLE
Getting the wrong amount of putrid wood when crafted

### ISSUE THREAD

**Issue by Tsun1ko** [mim1q/MineCells#131]
In vanilla minecraft most wood blocks give 3 when crafted but putrid wood only gives 1

**Comment by mim1q** [comment]
Right! I remembered to add the recipe but forgot to tweak the resulting amount 😅 
Fixing that right up!
